projects/addon-doc/src/components/tab-content/tab-content.template.less

```diff
@@ -0,0 +1,34 @@
+@import 'taiga-ui-local';
+
+@keyframes fadeIn {
```

Comment:
I believe we have it somewhere in global styles already

Reply:
Do you mean use `tuiFadeIn` directly or use `[@tuiFadeIn]` in template?